<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>687</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>561</width>
    <height>687</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>561</width>
    <height>687</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>华东交通大学</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="checkBox_advance">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>高级选项</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_excel_file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>521</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>选择 Excel 文件</string>
    </property>
    <widget class="QGroupBox" name="groupBox_excel_info">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>461</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Excel 内容选取</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>413</width>
        <height>107</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="cur_excel_name_label">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>请选择Excel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QComboBox" name="comboBox_excel_sheet">
         <property name="minimumSize">
          <size>
           <width>241</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>241</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>请先选择Excel文件</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>401</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>401</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;第一个成绩的行列&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QComboBox" name="comboBox_get_row">
         <property name="minimumSize">
          <size>
           <width>181</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>181</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>请先选择Excel</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QComboBox" name="comboBox_get_col">
         <property name="minimumSize">
          <size>
           <width>181</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>181</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>请先选择Excel</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>30</y>
       <width>466</width>
       <height>146</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" rowspan="4">
       <widget class="QListWidget" name="listWidget_excel_list">
        <property name="minimumSize">
         <size>
          <width>377</width>
          <height>144</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>377</width>
          <height>144</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_select_excel">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>选择</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer_3">
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>71</width>
          <height>21</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>68</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btn_re_view_excel">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btn_open_dir">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>打开文件夹</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_about_info">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>521</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>关于</string>
    </property>
    <widget class="QLabel" name="ecjtu_logo_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>ecjtu_logo_img</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_speace_insititute">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string> 华东交通大学 - 软件学院
 和平研究院</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton_feedback">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>关于</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>521</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>请到需要录成绩的页面再点击开始按钮</string>
    </property>
    <widget class="QPushButton" name="btn_click_checkbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>点击复选框</string>
     </property>
    </widget>
    <widget class="QComboBox" name="level_comboBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>31</y>
       <width>271</width>
       <height>39</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_start">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>开始录入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_clean_input">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>清空输入</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_advance">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>521</width>
      <height>91</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>高级</string>
    </property>
    <widget class="QPushButton" name="btn_clean_log">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>清空日志文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_re_open_browser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>重新打开浏览器</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_map">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>更改等级成绩映射</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
